services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/factory/ApplicationAttachmentFactory.kt

```diff
@@ -58,18 +58,25 @@ class ApplicationAttachmentFactory(
         return ApplicationAttachmentBuilder(entity, attachmentRepository, this)
     }
 
-    fun saveAttachment(applicationId: Long): ApplicationAttachmentEntity {
+    fun saveAttachmentToDb(applicationId: Long): ApplicationAttachmentEntity {
```

Comment:
There's nowhere else to save this than to the DB.

Reply:
I think I should have removed this method in #537 when I introduced the builder. Now there are several ways to create the same attachment. Maybe the uses of this method could be replaced with `save(application = application).withDbContent()`?

Reply:
Removed saveAttachmentToDb and saveAttachmentToCloud methods and replaced them with what you suggested.